tests/api/test_order.py

```diff
@@ -3016,3 +3039,93 @@ def test_update_invoice_invalid_id(user_api_client, permission_manage_orders):
     errors = content["data"]["updateInvoice"]["invoiceErrors"][0]
     assert errors["code"] == InvoiceErrorCode.NOT_FOUND.name
     assert errors["field"] == "id"
+
+
+def test_create_invoice(user_api_client, permission_manage_orders, orders):
+    number = "01/12/2020/TEST"
+    url = "http://www.example.com"
+    order = orders[0]
+    variables = {
+        "orderId": graphene.Node.to_global_id("Order", order.pk),
+        "number": number,
+        "url": url,
+    }
+    user_api_client.user.user_permissions.add(permission_manage_orders)
+    response = user_api_client.post_graphql(CREATE_INVOICE_MUTATION, variables)
+    content = get_graphql_content(response)
+    invoice = Invoice.objects.get(order_id=order.pk, url=url, number=number)
+    assert invoice.url == content["data"]["createInvoice"]["invoice"]["url"]
+    assert invoice.number == content["data"]["createInvoice"]["invoice"]["number"]
+    assert invoice.status == InvoiceStatus.READY
+
+
+def test_create_invoice_invalid_id(user_api_client, permission_manage_orders):
+    variables = {
+        "orderId": graphene.Node.to_global_id("Order", 1337),
+        "number": "01/12/2020/TEST",
+        "url": "http://www.example.com",
+    }
+    user_api_client.user.user_permissions.add(permission_manage_orders)
+    response = user_api_client.post_graphql(CREATE_INVOICE_MUTATION, variables)
+    content = get_graphql_content(response)
+    errors = content["data"]["createInvoice"]["invoiceErrors"][0]
+    assert errors["code"] == InvoiceErrorCode.NOT_FOUND.name
+    assert errors["field"] == "orderId"
+
+
+def test_create_invoice_missing_params(
+    user_api_client, permission_manage_orders, orders
+):
+    order = orders[0]
+    number = "01/12/2020/TEST"
+    url = "http://www.example.com"
+    variable_set = [
+        {
+            "orderId": graphene.Node.to_global_id("Order", order.pk),
+            "number": number,
+            "url": None,
+        },
+        {
+            "orderId": graphene.Node.to_global_id("Order", order.pk),
+            "number": None,
+            "url": url,
+        },
+    ]
+    user_api_client.user.user_permissions.add(permission_manage_orders)
+
+    for variables in variable_set:
+        response = user_api_client.post_graphql(CREATE_INVOICE_MUTATION, variables)
```

Comment:
```suggestion
        response = user_api_client.post_graphql(CREATE_INVOICE_MUTATION, variables, permissions=[permission_manage_orders])
```
Insted of addting permisionss to user. 
If we missed this in other invoices test we should fix it too.